ormconfig.js

```diff
@@ -0,0 +1,31 @@
+const dotenv = require('dotenv');
+
+const envFile = process.env.ENVIRONMENT
+  ? `.env.${process.env.ENVIRONMENT}`
+  : '.env';
+dotenv.config({ path: envFile });
+
+
+// If .env wasn't provided then exit
+if (!process.env.PORT) {
+  console.error('==> Please check your .env');
+  process.exit(1);
+}
+
+module.exports = {
+  type: 'postgres',
+  host: process.env.TYPEORM_HOST,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+  port: Number.parseInt(process.env.TYPEORM_PORT || '3000'),
```

Comment:
I think the port should be a number right?
```suggestion
  port: Number.parseInt(process.env.TYPEORM_PORT || 3000),
```

Reply:
In this case, `parseInt` just accepts a string, this will throw a warning...